beets/ui/commands.py

```diff
@@ -708,6 +736,15 @@ def choose_match(self, task):
                     _, _, candidates, rec = autotag.tag_album(
                         task.items, search_id=search_id
                     )
+            elif choice in extra_ops.keys():
+                # Allow extra ops to automatically set the post-choice.
+                # TODO: currently action.MANUAL and action.MANUAL_ID are not
+                # properly handled
+                post_choice = extra_ops[choice](self, task)
+                if post_choice in (importer.action.SKIP, importer.action.ASIS,
+                                   importer.action.TRACKS,
+                                   importer.action.ALBUMS):
+                    return post_choice
```

Comment:
Sure; this makes sense. And not permitting `MANUAL` and `MANUAL_ID` seems OK, since these are really part of the UI more than they are part of the importer. In fact, an argument could be made for removing them from the `importer.action` enum and making them the sole purview of the `TerminalImportSession` component.